### PR TITLE
Replace "unavailable" ionicon with a fixed size

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed complements [JENKINS-67837](https://issues.jenkins.io/browse/JENKINS-67837) by addressing my comment. I exchanged the ellipse by an "x" with applied scaling, to not stretch the ellipse all over the page.

### Proposed changelog entries

* Replace the "unavailable ionicon" icon by a smaller "x".

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]
i think okay

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (daniel-beck) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:57]: Why is the title "Close"? Is that shown to users and can give the impression of a "Close Window" button?

Comment (NotMyFault) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:57]: Not necessarily any "x" shown somewhere refers to closing something, e.g. aborting builds.
I thought that it would fit fine here, because it's somewhat of obvious that something went wrong.
However, I can look for a different ionicon for sure :)

Comment (daniel-beck) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:57]: This is about the title attribute, not the choice of icon.

Comment (NotMyFault) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:57]: Oh, I read that wrong then. My bad, but no, this is not shown to users.

**Review by janfaracik — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
